### DISCUSSION TITLE
Change estimated image size moving image size slider

### DISCUSSION THREAD

**Discussion by andybuki** [jbaiter/pdiiif#19] [Ideas]
Hi, if i move the image size slider the estimated image size stays the same.
Do you plan to adapt it?

![image](https://user-images.githubusercontent.com/4548460/218280913-8f89ae9d-7637-4c67-b07f-e627498c5f86.png)

**Comment by andybuki** [comment]
perfect, in a new version it works

**Reply by jbaiter** [reply]
Sorry, yes, this was a bug that I introduced in an earlier change. The estimate should now update whenever any of the values that influence the file size (scale, choice of pages, manifest itself) change.
